Dshop/apps/products_catalogue/tests/test_api_products.py

```diff
@@ -100,71 +98,122 @@ def assert_product_with_images(data):
 
 
 @pytest.mark.django_db
-def test_product_detail_with_images(authenticated_api_client, create_product_with_images):
+def test_product_detail_with_images(api_client_authed, create_product_with_images):
     url = reverse('products-api-detail', kwargs={'pk': create_product_with_images.id})
-    response = authenticated_api_client.get(url)
+    response = api_client_authed.get(url)
```

Comment:
Why change api client?
Also, this endpoint should be available to not authenticated API client, could you please change?